packages/kaspersky-components/src/select/selectCss.ts

```diff
@@ -37,10 +37,6 @@ export const selectCss = css`
     }
   }
 
-  &.ant-select-show-search:not(.ant-select-multiple) .ant-select-clear {
```

Comment:
**suggestion**: @bloomou let's add story with `showSearch` and `allowClear` options?

Reply:
done